packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -35,6 +37,18 @@ if (pluginsFile.exists()) {
 
 gradle.getGradle().projectsLoaded { g ->
     g.rootProject.beforeEvaluate { p ->
+        p.subprojects { sp ->
+            def sanitizedPluginName = sp.name.replace(":", "");
+            if (object != null && object.plugins != null && object.plugins.android != null
+                    && object.plugins.android.name.contains(sanitizedPluginName)) {
+                def androidPluginBuildOutputDir = new File(flutterModulePath + File.separator + File.separator
+                        + "plugins_build_output" + File.separator + sanitizedPluginName);
+                if (!androidPluginBuildOutputDir.exists()){
```

Comment:
The existing code in this file appears to have a space before `{` in conditionals; this should match.